### ISSUE TITLE
The performance of trained model is pool in train set.

### ISSUE THREAD

**Issue by yiran-THU** [SURFZJY/EAST-caffe#4]
I have trained this model in ic15 with code in this repository.
After 69100 iteration training, the loss is reduced to 0.00333686.
>I1024 22:11:46.818552  4375 sgd_solver.cpp:105] Iteration 69180, lr = 0.001
>I1024 22:11:53.286249  4375 solver.cpp:218] Iteration 69190 (1.54621 iter/s, 6.46743s/10 iters), loss = 0.00333686
>I1024 22:11:53.286276  4375 solver.cpp:237]     Train net output #0: Loss_rbox = 0.00646377 (* 1 = 0.00646377 loss)
>I1024 22:11:53.286281  4375 solver.cpp:237]     Train net output #1: Loss_score = 0.00405699 (* 0.01 = 4.05699e-05 loss)

However, The performance of trained model is pool even in train set.
![ic](https://user-images.githubusercontent.com/16274710/67538786-b59dcb00-f712-11e9-8590-fca483e6bdb3.jpg)
